Bio-Tech/Yael.md

```diff
@@ -0,0 +1,63 @@
+# Glossary
+## Biotechnology
+
+This glossary was created collaboratively with Our Best Words TechnComm students. This Biotechnology section was worked on by:
+[Yael Lachman](https://www.linkedin.com/in/yael-lachman/)
+[Rachel Goldstuck](https://www.linkedin.com/in/rachel-goldstuck/)
+[Eitan Gavson](https://www.linkedin.com/in/eitan-gavson/)
+
+[A](#a) [B](#b) C [D](#d) E F H G [I](#i) J K L [M](#m) N O [P](#p) Q [R](#r) S [T](#t) U V W X Y Z
+
+### A
+<dl>
+  <dt>autoimmune disease</dt>
+  <dd>a disease type in which the immune system can’t differentiate between its own cells and those which have invaded the body.</dd>
+</dl>
+
+### B
+<dl>
+  <dt>benign tumor</dt>
+  <dd>a collection of abnormal non-cancerous tissue cells that can grow uncontrollably and invade nearby tissues.</dd>
+</dl>
+
+### D
+<dl>
+  <dt>diigital health</dt>
+  <dd>the concept of providing medical treatments and information using technology.</dd>
+</dl> 
+
+### I
+<dl>
+  <dt>immunosuppressants</dt>
+  <dd>a type of medication that weaken the immune system, to prevent it causing harm to parts of the body that are needed, but that it may not recognize e.g. after an organ transplant.</dd>
```

Comment:
```suggestion
  <dd>A type of medication that weakens the immune system's response to foreign matter. Immunosuppressants prevent the body's immune system from attacking organs that it may not recognize as belonging to itself, such as after an organ transplant.</dd>
```